Sound/Tidal/Pattern.hs

```diff
@@ -842,6 +842,7 @@ d1 $ sound (samples "xx(3,8)" (tom $ choose ["a", "e", "g", "c"]))
 plays a melody randomly choosing one of the four notes \"a\", \"e\", \"g\", \"c\".
 -}
 choose :: [a] -> Pattern a
+choose [] =  throw (ErrorCall "Empty list. Nothing to choose from.")
```

Comment:
@pd3v :
  - You can/should just write ```error "Empty list. Nothing to choose from."``` (no ```throw``` or ```ErrorCall```)
  - The other lines are probably appearing in the diff because your editor/IDE are trimming trailing newlines